rwengine/src/audio/SoundManager.cpp

```diff
@@ -38,6 +38,10 @@ SoundManager::SoundManager(GameWorld* engine) : _engine(engine) {
 }
 
 SoundManager::~SoundManager() {
+    sounds.clear();
```

Comment:
Here and below is unneeded. (Map does on its own.)

Reply:
You need to delete the openal buffers inside the sound objects before openAL closes.

Reply:
Or maybe even better - wrapping in function?

Reply:
Removing these 3 clears, will still lead to the warning at program shutdown.

Reply:
I mean removing `sfx.clean()` and moving rest into new function.